### DISCUSSION TITLE
Template Import Failed

### DISCUSSION THREAD

**Discussion by benlabbeus** [n3wt0n/AzureWebAppSSLManager#33] [Q&A] [answered]
Failed with a conflict on site. Tried a couple times with different names, same error. Any help much appreciated!

<img width="717" alt="Screen Shot 2021-03-03 at 10 41 29 PM" src="https://user-images.githubusercontent.com/5489053/109922089-b4b76a80-7c71-11eb-9724-7f8b6e8780fb.png">

**Comment by n3wt0n** [comment, marked as answer]
Hey, sorry forgot about this 🙏🏻

This is related to issue #38 

When I renamed the branch from `Master` to `Main`, I didn't change it in the deployment script so after creating the resources it was still trying to get the code from `master`. Now it is fixed

**Comment by rfreytag** [comment]
Are pull requests welcome?  

As an Azure noob that doesn't use Github much I had to learn a lot to get AzureWebAppSSLManager working.  I have suggestions on how to make it an easier start for your less proficient Azure admins if they are welcome?
 
-----Original Message-----

**Reply by n3wt0n** [reply]
Yes, sure. Open the PR and then we can discuss about it there :)
